mule-extensions-api/src/main/java/org/mule/runtime/extension/api/annotation/param/RefName.java

```diff
@@ -30,8 +30,8 @@
  * @since 1.0
  */
 @Target(value = FIELD)
-@Retention(RetentionPolicy.RUNTIME)
+@Retention(RUNTIME)
 @Documented
-public @interface ConfigName {
+public @interface RefName {
```

Comment:
update jdoc =>   * with the name that the config has received on the application configuration.